### PR TITLE
Add bursaries variable

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/QkVF6jUs

### Context
We've recently introduced variables for financial figures across the website. This is so it's easier to update and manage those figures when they change.

### Changes proposed in this pull request
Bursary and scholarship funding amounts are only applicable for a certain time period, then they may change. This PR turns that time period into a variable so we can easily update it.

### CODE REVIEW

File: config/values/bursaries.yml [hunk 1]
```diff
@@ -19,3 +19,4 @@ bursaries:
     physics: £28,000
     religiouseducation: £10,000
     nonuk: £28,000
+  fundingwindow: between September 2024 and July 2025
```
Comment (gemmadallmandfe) [inline config/values/bursaries.yml:22]: Looks good!

My only (very minor) comment would be whether to simplify the variable name eg `bursaries_fundingdates` or `bursaries_dates` but up to you

File: app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb [hunk 2]
```diff
@@ -21,7 +21,7 @@
     link_title: "Learn more about fees and financial support for non-UK trainee teachers.",
     link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
   ) %>
-    <p>The figures on this page apply if you're starting your training between September 2024 and July 2025.</p>
+    <p>The figures on this page apply if you're starting your training <%= value :bursaries_fundingwindow %>.
```
Comment (emmaachesongray) [inline app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb:24]:
```suggestion
    <p>The figures on this page apply if you're starting your training <%= value :bursaries_fundingdates %>.
```

File: config/values/bursaries.yml (hunk 1 above)
Comment (emmaachesongray) [inline config/values/bursaries.yml:22]:
```suggestion
  fundingdates: between September 2024 and July 2025
```

File: app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb [hunk 3]
```diff
@@ -21,7 +21,7 @@
     link_title: "Learn more about fees and financial support for non-UK trainee teachers.",
     link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
   ) %>
-    <p>The figures on this page apply if you're starting your training between September 2024 and July 2025.</p>
+    <p>The figures on this page apply if you're starting your training <%= value :bursaries_fundingdates %>.
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb:24]: Just noticed that you've deleted the closing p tag - which doesn't seem right @emmaachesongray ?

When done, do you want to get Martyn to give a 2i and then we can merge it?

Comment (emmaachesongray) [reply app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb]: fixed that one now and will message Martyn :)

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by martyn-w — APPROVED** [review]
👍 LGTM

**Pull request merged**